Release Notes 4.1.0 (#1488)
dotnet/SqlClient: release-notes/4.1/4.1.0.md

# Release Notes

## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022

This update brings the below changes over the previous preview release:

### Added

> - Added New Attestation Protocol `None` for enclave type `VBS` with optional `AttestationUrl`. This connection string property when set will allow users to forgo enclave attestation for VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425)

Reviewer: ```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425)
```

# Release Notes

> ## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022

Reviewer 2: ```suggestion
## Microsoft.Data.SqlClient 4.1.0 released 31 January 2022
```

This update brings the below changes over the previous preview release:

### Added

> - Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)

Author: ```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#introduce-attestation-protocol-none)
```

> - Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)

Author: ```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
```

> ### Use Attestation Protocol None to skip enclave attestation for VBS enclaves

Author: ```suggestion
### Introduce Attestation Protocol None
```

> ### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.

Author: ```suggestion
### Use Attestation Protocol None to skip enclave attestation for VBS enclaves
```

A new attestation protocol called `None` will be allowed in the connection string. This protocol will allow users to forgo enclave attestation for `VBS` enclaves. When this protocol is set, the enclave attestation URL property is optional.  

> Connection string examples:

Reviewer: ```suggestion
Connection string example:
```
